microbench/bfs/include/pando-bfs-galois/sssp.hpp

```diff
@@ -285,11 +290,20 @@ pando::Status SSSPMDLCSR(G& graph, std::uint64_t src, HostLocalStorage<MDWorkLis
         auto [srcID, found] = graph.getLocalTopologyID(src);
         if (found) {
           graph.setDataOnly(srcID, 0);
-          PANDO_CHECK(fmap(toRead[0], pushBack, srcID));
+
+          std::int16_t srcHost = graph.getPhysicalHostID(src);
+          if (srcHost == pando::getCurrentPlace().node.id) {
```

Comment:
this is wrong.

Reply:
Why? If I don't add that line, one of the test will break.
You should only add the master of the source vertex to the worklist but not the mirror in OEC.

Reply:
oh i see no you are right.